doc/user/content/ops/troubleshooting.md

```diff
@@ -302,3 +302,160 @@ SELECT DISTINCT
 FROM dependencies D
 JOIN mz_objects O ON (D.import_id = O.id);
 ```
+
+## Why isn't my source ingesting data?
+
+First, look for errors in [`mz_source_statuses`](/sql/system-catalog/mz_internal/#mz_source_statuses):
+
+```sql
+SELECT * FROM mz_internal.mz_source_statuses
+WHERE id = <SOURCE_ID>;
+```
+
+You can get your source's id with:
+
+```sql
+SELECT id FROM mz_sources
+WHERE name = '<your source name>';
+```
+
+If your source reports a status of `stalled` or `failed`, you likely have a
+configuration issue. The returned `error` field will provide details.
+
+If your source reports a status of `starting` for more than a few minutes,
+[contact support](/support).
+
+If your source reports a status of `running`, but you are not receiving data
+when you query the source, the source may still be ingesting its initial
+snapshot. See the next section.
+
+## Has my source ingested its initial snapshot?
+
+Query the `snapshot_comitted` field of the
+[`mz_source_statistics`](/sql/system-catalog/mz_internal/#mz_source_statistics) table:
+
+```sql
+SELECT bool_and(snapshot_committed) as snapshot_committed
```

Comment:
Materialize tells me:
```
ERROR:  bool_and not yet supported
```

Reply:
This works for me I think:

```sql
SELECT
    sources.id,
    sources.name,
    MIN(snapshot_committed::int)::bool as snapshot_committed
FROM
    mz_internal.mz_source_statistics stats
    JOIN mz_catalog.mz_sources sources
        ON sources.id = stats.id
GROUP BY sources.id, sources.name;
```

Reply:
You have to be a recent main!

Reply:
I don't think we want to promote `MIN` to do this!

Reply:
We will merge this pr after the release